blockchain/processing.py

```diff
@@ -255,6 +299,8 @@ def _execute_phase_1(self, config, current_block_id):
             # update status of approved transactions
             for tx in approved_transactions:
                 tx["header"]["status"] = "approved"
+                if tx["header"]["transaction_type"] == "TT_SUB_REQ":
```

Comment:
Yes - I did, and I **love** the idea.
We'll want to think about the design so that it doesn't confuse (my subscription to your transaction type `TT_X` would have a custom smart contract on my node that would be different than your smart contract - should they be held separately as subscription smart contracts, and is there ever a need to do the same for received backlog VR records (I think there is).

We should talk in person next week.